visualization/open3d_vis_utils.py

```diff
@@ -67,7 +67,7 @@ def create_colored_spheres_open3d(
 
 
 def create_all_frustums_open3d(wTi_list: List[Pose3], calibrations: List[Cal3Bundler]) -> List[open3d.geometry.LineSet]:
-    """Render camera frustums as collections of line segments, using Open3d.
+    """Render camera frustums as collections of line segments, using Open3d. Colored red-to-green by trajectory order.
```

Comment:
What is trajectory order for unordered collections?

Reply:
Trajectory order for unordered collections would just be the order by which the images are numbered, when we read the from disk.

Reply:
I added a comment to explain this a bit more @akshay-krishnan